ibm_mq/datadog_checks/ibm_mq/collectors/metadata_collector.py

```diff
@@ -0,0 +1,53 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.base.utils.common import to_native_string
```

Comment:
Please use `to_string` instead for this integration, some users still need to install it on older agent versions.